web/tests/drafts.test.cjs

```diff
@@ -8,14 +8,15 @@ const {run_test, noop} = require("./lib/test.cjs");
 const $ = require("./lib/zjquery.cjs");
 
 const user_pill = mock_esm("../src/user_pill");
+const settings_data = mock_esm("../src/settings_data");
```

Comment:
Long term, we should be aiming to just require the actual module having set up an `example_realm` module like `example_user.ts`, that does default values for all the fields and is type-correct. But obviously mocking this is still much better than rewiring, so again, not a blocker for merging.